data/human/transport missions.txt

```diff
@@ -3582,3 +3582,286 @@ mission "Hauler VI cargo prototype 3"
+			`	On your way out of the room, you notice a quickly scrawled note on top of a chair. It reads "North, Ultima Thule," leaving the reader to decide what it entails.`
+	
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+			confusion 20
+		system "Alpha Hydri"
+		fleet "Small Northern Pirates"
+
+mission "James Medicine [1] Preventive"
+	# Name is important; it should be lexically ordered after "James Medicine [1]"
+	invisible
+	landing
+	source "Nimbus"
+	to offer
+		has "James Medicine [0]: done"
+		has "james medicine: infected"
+	on offer
+		dialog
+			`Thinking about what the doctor said, it might be wise to buy some of the medicine for yourself. Checking the receipt you got, this would cost you 10,000 credits.`
+	on accept
+		payment -10000
+		set "james medicine: medicine taken"
+		fail
+
+mission "James Medicine [Fail disabled]"
+	invisible
+	landing
+	to offer
+		or
+			has "James Medicine [0]: failed"
+			has "James Medicine [0]: declined"
+			has "James Medicine [1]: failed"
+			has "James Medicine [1]: declined"
+		has "james medicine: infected"
+	on offer
+		set "james medicine: james disabled"
+		fail
+
+mission "James Medicine [Fail sick]"
+	invisible
+	landing
+	to offer
+		and
+			not "James Medicine [0]: active"
+			not "James Medicine [1]: active"
+		has "james medicine: infected"
+	on offer
+		event "james medicine sick" 60 100
+		fail
+
+event "james medicine sick"
+
+mission "James Medicine [2]"
+	invisible
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
+		near "Sol" 100
+		attributes spaceport
+	to offer
+		has "event: james medicine sick"
+	on offer
+		conversation
+			`As you gently lower your ship into <origin>'s atmosphere, you begin to feel tired, starting from the tips of your fingers and quickly spreading to your whole body. By the time you take notice of it, you are already drowsy and realize you might be unable to land before you fall completely asleep.`
+			branch medicineless
+				not "james medicine: medicine taken"
+			`	In the height of the moment, you remember that when you were buying medicine for James, you bought some for yourself as well. You reach for the box and open it. Inside are a dozen or so neatly-arranged amber-colored pills and a long list of instructions shouting "EXPERIMENTAL" and "UNTESTED" in bold red letters.`
+			choice
+				`	(Take a pill.)`
+				`	(Think of other solutions.)`
+					goto medicineless
+			`	You hastily take one of the pills and swallow it, as per the instructions on the box. You can already start feeling better, even though the dizziness does not go away immediately. Just in case, you switch the autopilot on. The last thing you see is the ship's display reading "Landing on <origin>" as you briefly close your eyes.`
+			`	When you wake up, spaceport workers on <origin> have just started refueling your ship.`
+				accept
+			label medicineless
+			branch crewless
+				"flagship crew" < 1
```

Comment:
You might want to consider changing the number here to more than one, since you imply later during the crew conversation that there are multiple crew members besides youself- enough, in fact, that "a few" is a portion of the crew and not the entire crew itself.